### DISCUSSION TITLE
Remove white margins

### DISCUSSION THREAD

**Discussion by Yanis-C** [sumatrapdfreader/sumatrapdf#2472] [Q&A]
Hello, i wanted to know if it's possible to remove white margins while reading, because on some pdf (ebooks) it makes the principal images a bit moved

Thanks !

**Comment by GitHubRulesOK** [comment]
the marginal aspect of pdf or ebook is set by the author thus changing that usually requires altering the source document. with PDF its not always possible if the "white" space is part of the image hence fit content width will be variable fom pdf to pdf, with ebooks it may be easier if there are no full width images,

The following is based on current FixedUI when reading an ePub (uses the normal zoom functions.)
The behavior of fit content can sometimes seem odd, first ensure you are in continuous mode to be applied to all pages, pick any page that is short but full width in content then select zoom fit content will work for the whole document, however if you fit content on a portrait cover image then it will naturally have whitespace in fuller width the best fit page is often the copyright page :-)
![image](https://user-images.githubusercontent.com/12754913/162641566-03312062-d313-4dfc-9014-dc103a1ba4c6.png)
